src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -74,3 +74,10 @@
 "Value"=dword:00000000
 "Title"="Force use of runtime code generation for Razor (requires restart)"
 "PreviewPaneChannels"="IntPreview,int.main"
+
+// CacheTag value should be changed when registration file changes
+// See https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/39345/Manifest-Build-Deployment-and-Setup-Authoring-In-Depth?anchor=example-pkgdef-key for more infomation
+[$RootKey$\SettingsManifests\{13b72f58-279e-49e0-a56d-296be02f0805}]
+@="Microsoft.VisualStudio.RazorExtension.RazorPackage"
+"ManifestPath"="$PackageFolder$\UnifiedSettings\razor.registration.json"
+"CacheTag"=qword:5DE8496A8900B809
```

Comment:
the wiki said this is updated on f5 scenarios, but I don't know if it still needs to be updated every time we change anything. 

```
"CacheTag": qword - A 64-bit integer value used to indicate changes to the content of the Settings Manifest

The first time a given Manifest Path is discovered, the manifest is parsed and the contents cached. The value of the cache tag is recorded. On subsequent runs of VS the value of "CacheTag" is compared to the one recorded. If the values do not match, then the associated entries in the cache are invalidated, and the manifest is re-processed.

When using the recommended 'ProvideSettingsManifest` attribute, this value is updated on every build using the long ticks of DateTime.Now.
```

I suggest testing for future people reading this just in case it needs to be manually updated